src/TextUI/TestRunner.php

```diff
@@ -941,6 +941,10 @@ private function handleConfiguration(array &$arguments): void
                 $this->addExtension($extension->createHookInstance());
             }
 
+            foreach ($arguments['subscribers'] as $subscriber) {
+                // TODO - how to register new subscribers?
```

Comment:
This will be done via our Events\Facade.

Right now, you could query the Facade for the Dispatcher (`Events\Facade::dispatcher()`) and call `register()` to add subscribers.

This is work in progress and the Facade will soon provide a direct API without leaking the Dispatcher.